src/gbd_mapping/__init__.py

```diff
@@ -1,7 +1,18 @@
-from .id import me_id, rei_id, c_id, s_id, cov_id, hs_id, scalar, UNKNOWN, UnknownEntityError
-from .base_template import GbdRecord, ModelableEntity, Restrictions, Tmred, Categories
+from ._version import __version__
+from .base_template import Categories, GbdRecord, ModelableEntity, Restrictions, Tmred
 from .cause import Cause, causes
-from .sequela import Sequela, Healthstate, sequelae
+from .covariate import Covariate, covariates
 from .etiology import Etiology, etiologies
+from .id import (
```

Comment:
How is this just now getting picked up??

Reply:
black or isort